src/stories/charts/ChartContainer.tsx

```diff
@@ -1,11 +1,11 @@
 import styled from "styled-components";
 
 interface ChartContainerProps {
-  width: number;
-  height: number;
+  width?: string;
+  height?: string;
 }
 
 export const ChartContainer = styled.div<ChartContainerProps>`
-  height: ${(props) => `${props.height}px` || "300px"};
-  width: ${(props) => `${props.width}px` || "100%"};
+  height: ${(props) => `${props.height}` || "300px"};
+  width: ${(props) => `${props.width}` || "100%"};
 `;
```

Comment:
è inutile questo tipo di controllo.  L'interfaccia mi dice che sono valori opzionali ma non ho un default. Ti faccio un esempio:
IF props.height is `"ciolla"`
`${props.height}` ===> true
else if props.height is `undefined`
`${props.height}` ===> true

Fix suggerito:
```
export const ChartContainer = styled.div<ChartContainerProps>`
  height: ${({ height }) => height || "300px"};
  width: ${({ width }) => width || "100%"};
`;
```